syntheseus/tests/reaction_prediction/chem/test_utils.py

```diff
@@ -0,0 +1,19 @@
+from rdkit import Chem
+
+from syntheseus.reaction_prediction.chem.utils import (
+    remove_atom_mapping,
+    remove_atom_mapping_from_mol,
+)
+
+
+def test_remove_mapping() -> None:
+    smiles_mapped = "[OH:1][CH2:2][c:3]1[cH:4][n:5][cH:6][cH:7][c:8]1[Br:9]"
+    smiles_unmapped = "OCc1cnccc1Br"
+
+    assert remove_atom_mapping(smiles_mapped) == smiles_unmapped
+    assert remove_atom_mapping(smiles_unmapped) == smiles_unmapped
+
+    mol = Chem.MolFromSmiles(smiles_mapped)
+    remove_atom_mapping_from_mol(mol)
+
+    assert Chem.MolToSmiles(mol) == smiles_unmapped
```

Comment:
If we're talking about this specific test, then there's not much that can happen here; removing the atom mapping is just a matter of clearing a certain atom property, it doesn't really matter how the molecule looks like or if the mapping was at all valid. The `remove_atom_mapping*` functions don't claim to return a _canonical_ molecule either, they just return something without the mapping, and canonicalization is handled elsewhere.

Reply:
I was just wondering if there are any edge cases where it can go wrong: for example, where the unmapping version will could accidentally alter the graph structure of the molecule, change chirality, etc. If the function is basically just removing numbers and you think nothing can go wrong then I think the test is enough.